MultiQuestensions/src/Beatmaps/PreviewBeatmapPacket.cpp

```diff
@@ -1,12 +1,21 @@
 #include "Beatmaps/PreviewBeatmapPacket.hpp"
+#include "custom-types/shared/register.hpp"
+using namespace MultiplayerExtensions::Beatmaps;
 
-DEFINE_CLASS(MultiplayerExtensions::Beatmaps::PreviewBeatmapPacket);
+DEFINE_TYPE(MultiplayerExtensions::Beatmaps, PreviewBeatmapPacket);
 
 namespace MultiplayerExtensions::Beatmaps {
+	std::vector<Il2CppClass*> GetPBPInterfaces() {
```

Comment:
Consider making this return a constexpr std::initializer_list instead of a vector, avoids a double allocation.

Also consider making a static local if not.